### PR TITLE
fix for binary and hexa edge case

### PR DESCRIPTION
**Pull request by iannottivictoria**
This fix is needed before the tests will work.
Handled the edge case of n=0 for binary and hex functions.

### CODE REVIEW

File: src/main/java/com/ecse437/project/services/Hexa.java [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 public class Hexa implements Function{
 	public String compute(int n) {
+		 if (n == 0) {
+			 return "0";
```
Comment (ryadammar1) [inline src/main/java/com/ecse437/project/services/Hexa.java:6]: Should be 0x0

File: src/main/java/com/ecse437/project/services/Main.java [hunk 2]
```diff
@@ -16,7 +16,7 @@ public static String generateAlgorithmOutput(Algorithm algorithm) throws Illegal
 			n = rand.nextInt(0, 11); // Past 10 is where the factorial values tend to get way too big
 			return computeResultAndRuntime(n, new Factorial());
 		  case Prime:
-			n = rand.nextInt(0, 512);
+			n = rand.nextInt(2, 512);
```
Comment (ryadammar1) [inline src/main/java/com/ecse437/project/services/Main.java:19]: Can we handle 0 and 1 in the Prime class instead of limiting the inputs?

**Review by ryadammar1 — CHANGES_REQUESTED** [review]

File: src/main/java/com/ecse437/project/services/Main.java (hunk 2 above)
Comment (iannottivictoria) [reply src/main/java/com/ecse437/project/services/Main.java]: Yes, changing the prime function.

File: src/main/java/com/ecse437/project/services/Hexa.java (hunk 1 above)
Comment (ryadammar1) [reply src/main/java/com/ecse437/project/services/Hexa.java:6]: Oops youre right thx for noticing

**Review by ryadammar1 — APPROVED** [review]
